jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -2524,7 +2524,101 @@ public boolean startsWith(String prefix) {
 	public boolean startsWith(String prefix, int start) {
 		return regionMatches(start, prefix, 0, prefix.lengthInternal());
 	}
+	
+/*[IF Java11]*/
+	/**
+	 * Strip leading and trailing white spaces from a string.
+	 * 
+	 * @return a substring of the original containing no leading
+	 * or trailing white spaces.
+	 */
+	public String strip() {
+		return stripSub(true, true);
+	}
+	
+	/**
+	 * Strip leading white spaces from a string.
+	 * 
+	 * @return a substring of the original containing no leading
+	 * white spaces.
+	 */
+	public String stripLeading() {
+		return stripSub(true, false);
+	}
+	
+	/**
+	 * Strip trailing white spaces from a string.
+	 * 
+	 * @return a substring of the original containing no trailing
+	 * white spaces.
+	 */
+	public String stripTrailing() {
+		return stripSub(false, true);
+	}
+	
+	/**
+	 * Strip leading and/or trailing white spaces from a string
+	 * 
+	 * @param set to true if leading white spaces should be removed,
+	 * otherwise leading spaces are ignored.
+	 * @param set to true if trailing white spaces should be removed,
+	 * otherwise trailing spaces are ignored.
```

Comment:
I think we need names after the `@param` clause. Also need to remove the ending period as per [1].

[1] http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#@param